dev/Kozani/KozaniManagerRuntime/M.K.MR.ManagerRuntimeManager.h

```diff
@@ -3,16 +3,25 @@
 
 #pragma once
 
-#include "Microsoft.Kozani.ManagerRuntime.ManagerRuntimeManager.g.h"
+#include <KozaniManager_h.h>
+#include "ManagerRuntimeManager.g.h"
 
 namespace winrt::Microsoft::Kozani::ManagerRuntime::implementation
 {
     struct ManagerRuntimeManager : ManagerRuntimeManagerT<ManagerRuntimeManager>
     {
         ManagerRuntimeManager() = default;
 
-        static winrt::Microsoft::Kozani::ManagerRuntime::ManagerRuntimeManager Open();
-        void TODO_ReplaceMeWithRealContent();
+        void ActivateRemoteApplication(
+            Windows::ApplicationModel::Activation::ActivationKind activationKind,
+            winrt::hstring appUserModelId,
+            winrt::hstring connectionRdpFilePath,
+            winrt::hstring additionalSettingsFilePath,
+            Windows::ApplicationModel::Activation::IActivatedEventArgs args,
+            IInspectable statusCallback);
+
+    private:
+        wil::com_ptr<IKozaniManager> m_kozaniManager;
```

Comment:
You'll want to put in a static call and not implicitly get the manager

As expressed here the caller pattern is

```
ManagerRuntime::ManagerRuntimeManager manager;
manager.ActivateRemoteApplication(...);
```

Better to have the calling pattern

```
auto manager{ ManagerRuntime::ManagerRuntimeManager::Open(...) }
manager.ActivateRemoteApplication(...);
```

Leaves open the door for us to support more granular pre-activation work (e.g. establish connection w/o running any app, configure behavior, other odd things). Also easier to provide correct lifetiming - the former snippet implicitly creates the manager on 1st use, whereas the latter it's explicit as part of the act of acquiring a manager.

P.S. And no, that doesn't mean we need ActivateRemoteApplication() and other checks doing any sort of `!m_kozaniManager` null check before use. Since there's no way to instantiate a Manager w/o calling Open (or whatever we call it) m_kozaniManager is never null when someone gets their hands on it, but with no potential concurrency issue. And yes, no one _should_ call the same instance from 2 threads, but people do all kinds of crazy things like that - sometimes by accident, sometimes intentionally probing for errors, crashes, MSRC vulnerabilities, etc. The static Open pattern helps prevent that (mis)use inherent to the use.

Reply:
Updated to ManagerRuntime::ManagerRuntimeManager::Create(...)